cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/Committee.hs

```diff
@@ -79,9 +79,9 @@ runGovernanceCommitteeKeyGenHot
     $ writeLazyByteStringFile vkeyPath
     $ textEnvelopeToJSON (Just Key.ccHotVkeyDesc) vkey
 
-data SomeCommitteeKey f
-  = ACommitteeHotKey  (f CommitteeHotKey)
-  | ACommitteeColdKey (f CommitteeColdKey)
+data SomeCommitteeKey
```

Comment:
Note that removing the type parameter also aligns with how this kind of type is usually written, see e.g. https://github.com/IntersectMBO/cardano-api/blob/3bbdf8ae962d91602a4d8b0b416e90a0e8024943/cardano-api/internal/Cardano/Api/DeserialiseAnyOf.hs#L237